script/bin/auto.sh

```diff
@@ -48,8 +53,19 @@ start() {
   fi
 }
 
+startOnPending() {
+  updatePid
+  if [ -z "$pid" ]; then
+    java -Ddruid.mysql.usePingMethod=false -Xms512M -Xmx2048M -XX:PermSize=512M -XX:MaxPermSize=1024M -XX:+HeapDumpOnOutOfMemoryError -Xverify:none -cp "${CLASS_PATH}" org.dinky.Dinky
+    echo "FLINK VERSION : $FLINK_VERSION"
+    echo "........................................Start Dinky Successfully........................................"
+  else
+    echo "Dinky pid $pid is in ${PID_PATH}/${PID_FILE}, Please stop first !!!"
+  fi
+}
```

Comment:
Why add this implementation?

Reply:
What I'm thinking is that although you can use tail - f to view logs, it's more user-friendly for beginners to check the startup status in the foreground. For example, the current start looks like a start -- daemon, and then I thought of adding such a foreground start.

Reply:
Overall, it's like wanting to improve the experience, but in fact, I feel that the better startup is a start and start -- daemon. If you think it's okay, I can make necessary modifications. If the experience is not good, I will delete this section.

Reply:
I agree with your point of view.